packages/shared-types/src/order/index.ts

```diff
@@ -17,6 +17,8 @@ export enum OrderStatus {
 export type OrderItem = {
   id: string;
   ticket: EventTicket;
+  isUsed: boolean;
```

Comment:
instead of `isUsed` could we use `valid: boolean` ? Like this field will be used to determine if ticket can be validated (and ticket can be unvalidated for other reasons that just scanning it)